### DISCUSSION TITLE
Difference between readfish and the nanopore adaptive sequencer gui back-end?

### DISCUSSION THREAD

**Discussion by danchurch** [LooseLab/readfish#232] [Q&A] [answered]
Hello!

Thanks for your really interesting software. Can you explain to a novice the difference(s) between ReadFish and the adaptive sequencing gui that is provided in MinKnow? Other than the use of cli/gui interfaces, of course. 

I'm assuming the MinKnow gui is simply implementing the readUntil package, but I haven't really found documentation confirming this. If this is the case, I think this would mean that an approach that uses  MinKnow adaptive sequencing would be based on voltage signals rather than the basecalls, and that a ReadFish-approach is of course based on rapid basecalling. 

I've asked on the nanopore community site also, but haven't gotten a clear answer. 

Thanks!

**Comment by mattloose** [comment]
Hi,

ONT adaptive sampling also uses rapid base calling.

ReadFish enables some features that are not available in ONT's standard adaptive sampling but the two methods both work in broadly the same way.

I hope that helps.

**Reply by danchurch** [reply]
Okay, that clarifies a lot, thank you. But can you tell me the advantages of Readfish over the current ONT defaults? I guess, what are these features that are enabled in ReadFish that you mention? Why use it over the ONT package?

**Reply by mattloose** [reply, marked as answer]
Good question!

For many experiments you may not need to use ReadFish over the ONT package. But if you wish to do complex experiments with different panels on different barcodes, or changing what you are targeting during an experiment or being able to independently start adaptive sampling during a run you can only do this with ReadFish at this time.
